tests/integration/v2_api/test_docs_matrix_tail_pea.py

```diff
@@ -0,0 +1,77 @@
+import pytest
+from collections import OrderedDict
+from jina import Document, DocumentArray, Executor, Flow, requests
+
+
+@pytest.mark.timeout(5)
+@pytest.mark.parametrize('replicas_shards', [(1, 1), (2, 2)])
```

Comment:
```suggestion
@pytest.mark.parametrize('replicas, shards', [(1, 1), (2, 2)])
def test_sharding_tail_pea(nu,m_replicas, num_shards):
```

I think this works better and u do not need to separate